src/com/ebcitron/piggybank/Bank.java

```diff
@@ -0,0 +1,34 @@
+package com.ebcitron.piggybank;
+import java.util.*;
+import  java.text.DecimalFormat;
+
+
+public class Bank {
+
+
+    private ArrayList<Coin> coinArr = new ArrayList<Coin>();
+
+    DecimalFormat DF = new DecimalFormat("$###,###.00");
+
+    public Bank() {
```

Comment:
If you have an empty constructor, you don't need to write it out. Java will give you a default one if you don't provide one